ui/components/ui/icon/search-icon.component.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function SearchIcon({ className, size, color }) {
```

Comment:
Update to use [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)

```suggestion
export default function SearchIcon({ className = undefined, size, color }) {
```